### DISCUSSION TITLE
Do you have email notifications on?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#27] [Q&A] [answered]
Do you have email notifications on?

**Comment by TechStudent10** [comment]
Why do you want to know?

**Reply by KendallDoesCoding** [reply]
Just wondering?

**Comment by TechStudent10** [comment, marked as answer]
Can't say, sorry.

I feel like telling you if I have email notifications on or off could pose a security risk. This is the internet. I'm just being skeptical.

Sorry.
